src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/fees/FeesControllerTest.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.civil.controllers.fees;
+
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import uk.gov.hmcts.reform.civil.controllers.BaseIntegrationTest;
+import uk.gov.hmcts.reform.civil.service.FeesService;
+import uk.gov.hmcts.reform.fees.client.model.Fee2Dto;
+import uk.gov.hmcts.reform.fees.client.model.FeeVersionDto;
+import uk.gov.hmcts.reform.fees.client.model.FlatAmountDto;
+
+import java.math.BigDecimal;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public class FeesControllerTest extends BaseIntegrationTest {
+
+    private static final String FEES_URL = "/fees/ranges/";
```

Comment:
The constant can be renamed accordingly: e.g. FEES_RANGES_URLS or FEE_RANGES_URLS